update_stage.sh

```diff
@@ -7,4 +7,5 @@ git pull
 
 cd ./bridge_adaptivity
 
-docker-compose -f ./docker-compose-stage.yml restart
+make migrate
+make run-stage
```

Comment:
run-stage wrapping `up` command which is not the same as `restart`, one I missed something.

Reply:
This command can recreate containers for a service if it finds changes in images or configurations (you can read more here https://docs.docker.com/compose/reference/up/). 
And this we can use as restart containers that have changed (instead of using restarting command that stop and recreate all containers).

Reply:
Thanks for the clarification